src/components/ToolchainSection/ToolchainSection.jsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import { Box } from 'grommet';
+
+import { theme } from '../../utils/theme';
+import { Text } from '../../legos/typography/Text';
+
+const toolchainItems = [
+  { id: '1', label: 'React', icon: './assets/icons/reactIcon.svg' },
+  { id: '2', label: 'Vue', icon: './assets/icons/vueIcon.svg' },
+  {
+    id: '3',
+    label: 'React Native',
+    icon: './assets/icons/reactNativeIcon.svg',
+  },
+  { id: '4', label: 'Angular', icon: './assets/icons/angularIcon.svg' },
+  { id: '5', label: 'Flutter', icon: './assets/icons/flutterIcon.svg' },
+  { id: '6', label: 'Node.js', icon: './assets/icons/nodeJSIcon.svg' },
+  { id: '7', label: 'Ionic', icon: './assets/icons/ionicIcon.svg' },
+];
+
+export const ToolchainSection = () => (
```

Comment:
Consider making the toolchain the same as on designs. I guess the header is missing and technologies are not correctly alligned vertically. Kindly compare:

* Design
![image](https://user-images.githubusercontent.com/12502321/84645048-b20cbd80-af08-11ea-85ef-a787ba539782.png)

* Markup
![image](https://user-images.githubusercontent.com/12502321/84645014-a620fb80-af08-11ea-92ff-5c5f630b4b82.png)